com.unity.render-pipelines.universal/Editor/ScriptableRendererFeatureEditor.cs

```diff
@@ -3,9 +3,13 @@
 
 namespace UnityEditor.Rendering.Universal
 {
+    /// <summary>
+    /// Editor script for a <c>ScriptableRendererFeature</c> class.
+    /// </summary>
     [CustomEditor(typeof(ScriptableRendererFeature), true)]
     public class ScriptableRendererFeatureEditor : Editor
```

Comment:
Same here, should be private or internal.

Reply:
Leaving it as public as it might break users with custom versions.